massrobotics_amr_sender_py/README.md

```diff
@@ -33,3 +33,27 @@ On you local workspace:
 colcon test --packages-select massrobotics_amr_sender
 colcon test-result --verbose
 ```
+
+## Releasing a new version
```

Comment:
My bad memory tells me this is something done by the maintainers, so it's not very useful to mix with usage documentation. It should probably go elsewhere. 
Could you do me a favor and check other packages to see if it's common to include the release instructions in the package README?

Reply:
TBH most of the packages I've seen don't have a README, but I agree that this shouldn't be here. Actually, these are notes I took while testing commands

Reply:
Perhaps a separate file, like `MAINTAINER_README.md` ? I agree the instructions shouldn't be in the README, but I wouldn't just throw them away since they are useful indeed